task.go

```diff
@@ -251,8 +251,19 @@ func (t *task) Resize(ctx context.Context, w, h uint32) error {
 	return err
 }
 
+const (
+	NoPivotRoot              = "io.runc.option.no-pivot-root"
```

Comment:
Or at least for now, have the ones here take the value of the ones in the shim to avoid typos.

Could have an option package, but that may be a overkill. Not sure if there's a convention for this scenario in the go community